docs/app/components/navbar/navbar-demo.component.html

```diff
@@ -12,10 +12,15 @@ <h5>About</h5>
 
     <hc-tile>
         <h5>Example</h5>
-        <hc-navbar appIcon="./assets/CashmereAppLogo.svg" brandIcon="./assets/TriFlame.svg" user="Christine K." [homeUri]="undefined" [fixedTop]="false">
+        <hc-navbar appIcon="./assets/CashmereAppLogo.svg" brandIcon="./assets/TriFlame.svg" [homeUri]="undefined" [fixedTop]="false">
```

Comment:
I like the way we show off the responsive navbar but the "user" was removed, what if we had a second example navbar with the user?

Reply:
I actually only removed the user because there was so little space in the demo tile and everything was getting squeezed.  But I agree it's good to have it in there.  So I'll just put it back in.